Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -288,18 +288,26 @@ private void ProcessPointerMovement(ref PointerModel pointer, ExtendedPointerEve
             ProcessPointerMovement(eventData, currentPointerTarget);
         }
 
+        #if false
```

Comment:
I suppose you will want to remove that dead code, now that we know the new implementation works?

Reply:
Dafuq. Thanks for catching that. Forgot to remove that crap. Fixed.